armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -173,41 +172,27 @@ def _isTopDummyBlockPresent(self):
         blkLst = self.linked.a.getBlocks()
         if not blkLst[-1].hasFlags(Flags.DUMMY):
             runLog.warning(
-                "No dummy block present at the top of {0}! "
+                f"No dummy block present at the top of {self.linked.a}! "
                 "Top most block will be artificially chopped "
-                "to preserve assembly height".format(self.linked.a)
+                "to preserve assembly height"
             )
             if self._detailedAxialExpansion:
                 msg = "Cannot run detailedAxialExpansion without a dummy block at the top of the assembly!"
                 runLog.error(msg)
                 raise RuntimeError(msg)
 
-    def axiallyExpandAssembly(self, thermal: bool = False):
-        """Utilizes assembly linkage to do axial expansion
-
-        Parameters
-        ----------
-        thermal : bool, optional
-            boolean to determine whether or not expansion is thermal or non-thermal driven
-
-        Notes
-        -----
-            The "thermal" parameter plays a role as thermal expansion is relative to the
-            BOL heights where non-thermal is relative to the most recent height.
-        """
+    def axiallyExpandAssembly(self):
```

Comment:
this could use a more detailed doc string... its rather sparse now. How would i get it to do cold height -> hot height expansion vs a simple updated temperature? Referencing methods to be called before it would help @albeanth

Reply:
reading the commit message, perhaps it should be renamed to 

`axiallyExpandAssemblyHotDims` or something like that if thats more of what it does now.

Reply:
@albeanth Whoops, perhaps I merged too soon.